requirements.txt

```diff
@@ -44,6 +44,7 @@ MarkupSafe==1.0
 mccabe==0.6.1
 mywsgi==1.0.3
 more-itertools==4.2.0
+mypy>=0.761
```

Comment:
What do you think about this? Ideally this should be in requirements-dev (which we do not have).

Reply:
The other dev dependencies seem to be in here too so it seems consistent at least

Reply:
I agree that it'd be good to split the development and testing requirements out, I don't think that it needs to be done as part of this change.